include/irr/asset/IVirtualTexture.h

```diff
@@ -48,7 +63,27 @@ class IVirtualTexture : public core::IReferenceCounted, public IVirtualTextureBa
 
     uint32_t neededPageCountForSide(uint32_t _sideExtent, uint32_t _level)
     {
-        return (std::max(_sideExtent>>_level, 1u)+m_pgSzxy-1u) / m_pgSzxy;
+        return (((_sideExtent+(1u<<_level)-1u)>>_level) + m_pgSzxy-1u) / m_pgSzxy;
+    }
+
+    enum E_SAMPLER_TYPE
+    {
+        EST_FLOAT,
+        EST_INT,
+        EST_UINT,
+        EST_INVALID
+    };
+    E_SAMPLER_TYPE formatToSamplerType(E_FORMAT _format)
```

Comment:
inline static and public? or move to some utility in `irr/asset/format` ?

Reply:
did this move?

Reply:
i removed this (along with E_SAMPLER_TYPE) completely